doc/source/tutorial.rst

```diff
@@ -0,0 +1,1466 @@
+********
+Tutorial
+********
+
+Before You Begin
+================
+
+.. note::
+
+    NSLS-II deploys a free, public "sandbox" for trying the software in the
+    browser using Jupyter notebooks. There is no need to install any software.
+    Go to `https://try.nsls2.bnl.gov <https://try.nsls2.bnl.gov>`_.
+
+* You will need Python 3.5 or newer. From a shell ("Terminal" on OSX,
+  "Command Prompt" on Windows), check your current Python version.
+
+  .. code-block:: bash
+
+    python3 --version
+
+  If that version is less than 3.5, you must update it.
+
+  We recommend install bluesky into a "virtual environment" so this
+  installation will not interfere with any existing Python software:
+
+  .. code-block:: bash
+
+    python3 -m venv ~/bluesky-tutorial
+    source ~/bluesky-tutorial/bin/activate
+
+  Alternatively, if you are a
+  `conda <https://conda.io/docs/user-guide/install/download.html>`_ user,
+  you can create a conda environment:
+
+  .. code-block:: bash
+
+    conda create -n bluesky-tutorial "python>=3.5"
+    conda activate bluesky-tutorial
+
+* Install the latest versions of bluesky and ophyd. Also install the databroker
+  unless you plan to skip the sections about accessing saved data. Finally,
+  install ipython (a Python interpreter designed by scientists for scientists).
+
+  .. code-block:: bash
+
+     python3 -m pip install --upgrade bluesky ophyd databroker ipython
```

Comment:
Do you want to also include a conda install (if people are using a conda env).

Reply:
Sure